sycl/doc/extensions/supported/sycl_ext_intel_fpga_device_selector.asciidoc

```diff
@@ -0,0 +1,120 @@
+= sycl_ext_fpga_selector
```

Comment:
```suggestion
= sycl_ext_intel_fpga_device_selector
```

This should match the name of the extension.

Reply:
done